src/App.tsx

```diff
@@ -138,7 +141,18 @@ enum APIRequestStatus {
   'SUCCEEDED',
 }
 
-export default function App() {
+export default function AppWithProvider() {
+  return (
+    <Provider store={store}>
+      <App />
+    </Provider>
+  );
+}
```

Comment:
Wrapping the App in a `<Provider>` with a store provided allows Redux to be available to the entire application